lib/semian/sysv_integer.rb

```diff
@@ -0,0 +1,12 @@
+module Semian
+  module SysV
+    class Integer < Semian::Simple::Integer #:nodoc:
```

Comment:
At this point, isn't it easier to write this entire thing in C? You don't need the `unless` below anymore.